DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateCourses/Index.cshtml

```diff
@@ -0,0 +1,53 @@
+﻿@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.DelegateCourses
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@model DelegateCoursesViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/courseSetup.css")" asp-append-version="true">
+
+@{
+  ViewData["Title"] = "Delegate courses";
```

Comment:
@SteveJacksonSoft Bit of a blunt question, but what is the purpose of this page? Is it just so the users have an easier time selecting courses before moving to the Course Delegates page? If that is the case, why don't we want to get rid of the course drop down menu on the Course Delegates page?

Reply:
Fair question. In the [same Slack thread](https://softwire.slack.com/archives/C016Q1GR2G5/p1644393276194899?thread_ts=1644339810.552289&cid=C016Q1GR2G5) as mentioned in the previous question, Kevin said:

> Hmm. Gut feeling says we keep the Course dropdown on the existing course delegates page, yes, for easy switching.

I think this page was particularly motivated by a certain centre which uses admin fields extensively, and not being able to see these admin fields at the "choose course" level made things much harder for them.
There'll probably be some use cases where admins want to look at delegates for a particular set of courses, and being able to jump from one to another directly without having to change page and search or filter will save them time.

Reply:
@SteveJacksonSoft I might be misunderstanding here then, but would that not be satisfied by, instead of making this whole new precursor page, just adding a button over on the Course Setup page cards that says "View delegates on course" which takes you to the relevant course delegates page. I'm just getting suspicious at the amount of similarities between the two pages.

Reply:
It could be done like that I think, but it would require some broader changes to keep a good UX, and doing that would change the layout of the site from what users are used to, so I'm not sure it would be a popular approach.

If we're going to have a Delegates area, users will want to view users by course within that area. If we're going to have a course setup area, it makes sense for that to be a top level area of its own, rather than with the Delegates area. So having two pages has UX value even if they are very similar. And the fact that we're using components mitigates the duplicated-code cost.

Reply:
Okay, those points are fair. Just double checking we weren't going to be satisfied with a simple-seeming workaround. Thanks for the explanation.

Reply:
It's a good idea like, thanks for raising it!